src/cdi_ndi/NDI_CP_Eloss.hh

```diff
@@ -0,0 +1,128 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   cdi_ndi/NDI_CP_Eloss.hh
+ * \author Ben R. Ryan
+ * \date   2020 Jun 3
+ * \brief  NDI_CP_Eloss class definition.
+ * \note   Copyright (C) 2019-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#ifndef cdi_ndi_NDI_CP_Eloss_hh
+#define cdi_ndi_NDI_CP_Eloss_hh
+
+#include "NDI_Base.hh"
+#include "cdi/CPCommon.hh"
+#include "cdi/CPEloss.hh"
+#include "ds++/Assert.hh"
+#include "units/PhysicalConstexprs.hh"
+
+namespace rtt_cdi_ndi {
+
+//============================================================================//
+/*!
+ * \class NDI_CP_Eloss
+ *
+ * \brief Derived rtt_cdi::CPEloss class for tabular eloss.
+ *        This class implements the interface found in cdi/CPEloss.hh for
+ *        the case where CP energy loss data is in tabular form, retrieved
+ *        from NDI.
+ */
+//============================================================================//
+
+class NDI_CP_Eloss : public rtt_cdi::CPEloss, public NDI_Base {
+public:
+  typedef std::vector<double> sf_double;
+
+public:
+  //! Constructor (default gendir path)
+  NDI_CP_Eloss(const std::string &library_in, rtt_cdi::CParticle target_in,
+               rtt_cdi::CParticle projectile_in);
+
+  //! Constructor (overridden gendir path)
+  NDI_CP_Eloss(const std::string &gendir_in, const std::string &library_in,
+               rtt_cdi::CParticle target_in, rtt_cdi::CParticle projectile_in);
+
+  //! Disable default constructor
+  NDI_CP_Eloss() = delete;
+
+  //! Disable copy constructor
+  NDI_CP_Eloss(const NDI_CP_Eloss &) = delete;
+
+  // >>> ACCESSORS
+
+  //! Get a stopping power.
+  double getEloss(const double temperature, const double density, const double v0) const;
+
+  //! Query to see if data is in tabular or functional form (true)
+  static constexpr bool is_data_in_tabular_form() { return true; }
+
+  //! Get the name of the associated data file
+  inline std::string getDataFilename() const { return std::string(); }
```

Comment:
Is this a placeholder for now?

Reply:
Oops, leftover, nice catch. I don't think we need this info since we'll already have `NDI_GENDIR_PATH` or some such in a `config.h` file.